lib/engine/config/game/g_1846.rb

```diff
@@ -116,7 +116,7 @@ module G1846
   },
   "market": [
     [
-      "Closed",
+      "Closedblk",
```

Comment:
hm what the heck is this? i guess i never looked at it, but it should be "0blk" or we can do "0l" or since we're naming everything now "0black" and everything else should be "0brown" etc

Reply:
Changed to 0 in 9dc0c943 -- that makes more sense anyway since "0" is what's displayed in the actual market cell. Used "blk" for "black" since "b" was taken for "brown". Kinda made me laugh when I realized the "o" in "Closed" was why "orange" was used.